Makefile

```diff
@@ -1,15 +1,22 @@
-ROOT_DEST=/opt/timecard-app
+
+APP_NAME_LOWER=timecard-app
+APP_NAME_UPPER=Timecard-App
+
+ROOT_DEST=/opt/${APP_NAME_LOWER}
 ROOT_BIN=/usr/bin
 ROOT_APPS=/usr/share/applications/
 ROOT_ICONS=/usr/share/icons/hicolor/scalable/apps/
 
-USER_DEST=/home/${USER}/.timecard-app
+USER_DEST=/home/${USER}/.${APP_NAME_LOWER}
 USER_BIN=/home/${USER}/.local/bin/
 USER_APPS=/home/${USER}/.local/share/applications/
 USER_ICONS=/home/${USER}/.icons/hicolor/scalable/apps/
 
-DESKTOP_FILE=com.indeliblebluepen.timecard_app.desktop
-ICON_FILE=timecard.svg
+SHARE_PATH=share
+DESKTOP_FILE_PATH=${SHARE_PATH}/applications
+DESKTOP_FILE=com.codemouse92.timecard.desktop
+ICON_FILE_PATH=${SHARE_PATH}/icons
+ICON_FILE=com.codemouse92.timecard.svg
```

Comment:
From the looks of it, the name/path of the desktop app were moved around a bit and didn't get updated in here. Guessing that the migration to Flatpak took precedence over this, which makes sense.

Reply:
Good call.